StRoot/StEvent/StETofHeader.cxx

```diff
@@ -38,6 +38,10 @@ StETofHeader::StETofHeader()
 {
     mRocGdpbTs.clear();
     mRocStarTs.clear();
+    const size_t kNbGet4sInSystem = 1728;
+    mMissMatchFlagVec = std::vector<bool>( kNbGet4sInSystem, false );
+    const size_t kNbCountersInSystem = 108;
```

Comment:
From the usage of this constant `kNbCountersInSystem` here and in other places it would make sense to make it static class variable. Also true for `kNbGet4sInSystem`